src/components/Personal-Zone/Publisher/JobsList/jobsList.tsx

```diff
@@ -2,24 +2,37 @@ import * as React from 'react';
 
 import Typography from '@material-ui/core/Typography';
 
+import { observer, inject } from 'mobx-react';
+
 import { RamadAd } from '../../../../types/User';
 import styles from './jobsListStyle';
 import Job from './job';
+import { JobsStore } from '../../../../store/JobsStore';
```

Comment:
Can't we add a folder shortcut there? (i.e `@store/JobsStore`)

Reply:
no we dont have webpack here